Lib/_pycodecs.py

```
@@ -44,47 +44,82 @@
+    "utf_32_ex_decode",
+]
 
 import sys
 import warnings
 from _codecs import *
 
 
-def latin_1_encode( obj, errors='strict'):
-    """None
-    """
+def latin_1_encode(obj, errors="strict"):
```

Pylint (3.3.7)

[error] function already defined line 96

(E0102)